index.html

```diff
@@ -0,0 +1,347 @@
+                            fill="currentColor" />
+                    </svg>
+                </a>
+            </div>
+        </div>
+    </section>
+
+
+    <!-- //projetcts section -->
+    <section class="projects">
+        <div class="project-card">
+            <div class="project-image">
+                <img src="assets/images/project1.png" alt="">
+            </div>
+
+            <div class="project-description">
+                <h1 class="project-title">Multi-Post <br> Stories</h1>
+                <div class="details">
+                    <p>CANOPY </p>
+                    <div class="cercle"></div>
+                    <p>Back End Dev</p>
+                    <div class="cercle"></div>
+                    <p>2015</p>
+                </div>
+
+                <p class="detail-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+                <div class="used-programming-languages">
+                    <p>html</p>
+                    <p>css</p>
+                    <p>javascript</p>
+                </div>
+            </div>
+
+            <div class="project-button-link-container">
+                <a href="">
+                    See project
+                </a>
+            </div>
+        </div>
+
+        <div class="project-card">
+            <div class="project-image">
+                <img src="assets/images/project2.png" alt="">
+            </div>
+
+            <div class="project-description">
+                <h1 class="project-title">Multi-Post <br> Stories</h1>
```

Comment:
- [ ] I think we should only have one `h1` in the project as the super title others are subtitles Kindly fix this to all your project and when is a subtitle we use `h2,..` for good practice and clean codes